lib/composite_primary_keys.rb

```diff
@@ -61,6 +61,8 @@
 require 'active_record/connection_adapters/abstract/database_statements'
 require 'active_record/connection_adapters/abstract_adapter'
 require 'active_record/connection_adapters/postgresql/database_statements'
+require "active_record/connection_adapters/trilogy_adapter"
+require "active_record/connection_adapters/trilogy/database_statements"
```

Comment:
These two requires should be removed. The only reason to have them is to enable CPK to override code contained in them, but I see no such overrides. In addition, these requires would make everyone using CPK have to install the trilogy adapter which we definitely don't want to do.